client/js/controllers/game.js

```diff
@@ -54,13 +61,37 @@ angular.module('mean.system')
         }
       };
 
+      $scope.checkFriendDisable = selectedEmail =>
```

Comment:
I would prefer moving all the logic that has to do with the invite modal in a component
Check this branch for sth similar, the chatBox Component  "using_components"